### PR TITLE
other plugin for javax to jakarta

### PR DESCRIPTION
**Pull request by aamotharald**
this will get approved faster..

### CODE REVIEW

File: cf-java-logging-support-servlet-jakarta/pom.xml [hunk 1]
```diff
@@ -120,13 +134,14 @@
...
 				<artifactId>build-helper-maven-plugin</artifactId>
-				<version>3.3.0</version>
+				<version>${build-helper-maven-plugin.version}</version>
 				<executions>
 					<execution>
 						<id>add-generated-sources</id>
 						<goals>
 							<goal>add-source</goal>
 						</goals>
+						<phase>generate-sources</phase>
```
Comment (mofterdinger) [inline cf-java-logging-support-servlet-jakarta/pom.xml:144]: Is this phase configuration really required? I'm asking regarding the plugin documentation it should be already bound by default to this phase: https://www.mojohaus.org/build-helper-maven-plugin/add-source-mojo.html

File: cf-java-logging-support-servlet-jakarta/pom.xml [hunk 2]
```diff
@@ -138,49 +153,52 @@
 						<goals>
 							<goal>add-test-source</goal>
 						</goals>
+						<phase>generate-test-sources</phase>
```
Comment (mofterdinger) [inline cf-java-logging-support-servlet-jakarta/pom.xml:156]: Really required ? See the default binding: https://www.mojohaus.org/build-helper-maven-plugin/add-test-source-mojo.html

File: cf-java-logging-support-servlet-jakarta/pom.xml [hunk 3]
```diff
@@ -138,49 +153,52 @@
...
 						<configuration>
 							<sources>
 								<source>target/generated-test-sources/java</source>
 							</sources>
 						</configuration>
 					</execution>
+					<execution>
```
Comment (mofterdinger) [inline cf-java-logging-support-servlet-jakarta/pom.xml:163]: The folder `target/generated-test-sources/resources` is empty, everything is extracted to `target/generated-sources/java` and `target/generated-test-sources/java`. Not sure if this is really required.

Comment (mofterdinger) [reply cf-java-logging-support-servlet-jakarta/pom.xml:163]: ah no, the logback-text.xml is extracted to this folder. Forget my last comment.

File: cf-java-logging-support-servlet-jakarta/pom.xml (hunk 2 above)
Comment (aamotharald) [reply cf-java-logging-support-servlet-jakarta/pom.xml:156]: Hi @mofterdinger ,
you are right, the default binf´ding should be omitted for better readability.
I also have made some slight changes to execute in the (100% correct) maven build lifecycle phases, I'll make another PR ..

**Pull request merged**